packages/forms-web-app/src/controllers/full-appeal/type-of-planning-application.js

```diff
@@ -38,7 +38,15 @@ exports.postTypeOfPlanningApplication = async (req, res) => {
   const { appeal } = req.session;
 
   const selection = body['type-of-planning-application'];
-  appeal.beforeYouStartSection = { typeOfPlanningApplication: selection };
+
+  const updatedAppeal = {
+    ...appeal,
+    appealType: mapPlanningApplication(selection),
+    beforeYouStartSection: {
```

Comment:
Why do we need this section?

Reply:
This is what was there previously, I'm just adding the appeal type.